bigframes/session/_io/bigquery/read_gbq_table.py

```diff
@@ -152,24 +152,23 @@ def validate_table(
     return False
 
 
-def are_index_cols_unique(
+def infer_primary_key(
     bqclient: bigquery.Client,
     table: bigquery.table.Table,
     index_cols: List[str],
     api_name: str,
     metadata_only: bool = False,
-) -> bool:
-    if len(index_cols) == 0:
-        return False
+) -> Tuple[str, ...]:
```

Comment:
Let's do a docstring here:

```suggestion
) -> Tuple[str, ...]:
"""Return a set of columns that can provide a unique row key or empty if none can be inferred.

Note: primary keys are not enforced, but these are assumed to be unique
by the query engine, so we make the same assumption here.
"""
```